external/fv3fit/tests/emulation/test_keras.py

```diff
@@ -171,3 +172,22 @@ def test_StandardLoss_prepare_arbitrary_kwargs():
 
     model = _get_model(10, 2)
     config.compile(model)
+
+
+def test_get_model_output_sensitivities():
+    sample = {
+        "a": tf.random.normal((100, 5)),
+        "b": tf.random.normal((100, 5)),
+    }
+    sample["field"] = sample["a"] + sample["b"]
+
+    inputs = {"a": tf.keras.Input(5), "b": tf.keras.Input(5)}
+    out = tf.keras.layers.Lambda(lambda x: {"field": x["a"] + x["b"]})(inputs)
+    model = tf.keras.Model(inputs=inputs, outputs=out)
```

Comment:
Optional...but does it need to be a model? I thought this would work with a plain old function. This would also ensure `get_model_output_sensitivies` is not using any of the more arcane APIs of `tf.keras.Model`.
```
def model(x):
    return {"field": x["a"] + x["b"]}
```

Reply:
It did not need a model to work, so I switched to use this